diagrams-demo-gallery/package.json

```diff
@@ -9,10 +9,10 @@
 		"url": "https://github.com/projectstorm/react-diagrams.git"
 	},
 	"scripts": {
-		"start": "../node_modules/.bin/start-storybook",
-		"storybook:build": "../node_modules/.bin/build-storybook -c .storybook -o .out",
-		"github": "../node_modules/.bin/storybook-to-ghpages",
-		"test:run": "../node_modules/.bin/jest --no-cache",
+		"start": "npx start-storybook",
```

Comment:
i would much prefer to use the binaries that are already inside the node_modules bin folder.

Reply:
updated